config.js

```diff
@@ -1594,6 +1594,8 @@ var config = {
 
     // Tile view related config options.
     // tileView: {
+    //     // Tileview is disabled only if enabled flag is specifically set to false. If not specified, it is enabled.
+    //     enabled: true,
```

Comment:
Please consider renaming this to `disabled` or something similar to make the flag self explanatory